src/components/Todo.test.tsx

```diff
@@ -4,29 +4,49 @@ import Todo from "./Todo";
 import {
   mockAddTodo,
   mockAddTodoError,
+  mockDeleteTodo,
+  mockDeleteTodoError,
   mockFetchTodoList,
   mockFetchTodoListError,
 } from "../mocks/mock";
 import userEvent from "@testing-library/user-event";
 
 test("Has a submit button", () => {
-  render(<Todo fetchList={mockFetchTodoList} addTodo={mockAddTodo} />);
+  render(
+    <Todo
+      fetchList={mockFetchTodoList}
+      addTodo={mockAddTodo}
+      deleteTodo={mockDeleteTodo}
+    />
+  );
```

Comment:
Noticed there is some duplicated set up for the tests in this file. Can you think of a way to remove this duplication?

Reply:
Btw this is not blocking.

Reply:
I tried using a `beforeEach` but it wouldn't let me put the `render` method inside. I've tried doing some research but I'm having trouble finding another method to remove the duplicate components in this test file.